barbican/templates/db-maintain-configmap.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: db-maintain-barbican
+  labels:
+    system: openstack
+    type: configuration
+    component: database
+
+data:
+  db-maintaine.sh: |
```

Comment:
Are you sure? I think we are passing the info down to the postgres chart.
In our regional charts we have:
```
openstack:
  barbican:
    postgres:
      password: ***
```
This should end up as `.Values.password` in the postgres chart IMO.